pages/[storage].js

```diff
@@ -0,0 +1,59 @@
+import { useRouter } from 'next/router';
+import useLocalStorageState from 'use-local-storage-state';
+import Header from '../components/Header';
+import ItemList from '../components/ItemList';
+import AddForm from '../components/AddForm';
+
+export default function Storage({ onAddItem }) {
+  const router = useRouter();
+  const { storage } = router.query;
+
+  const [storages] = useLocalStorageState('storages');
+  const [items, setItems] = useLocalStorageState('items');
+
```

Comment:
Most of the time it's best to initialize arrays with empty arrays instead of `undefined` or `null`. I'm not 100% sure about your usecase, but you should consider that.